test/unittest/tests/test_session_config.cpp

```diff
@@ -0,0 +1,66 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include <catch2/catch.hpp>
+
+#include "wt_internal.h"
+#include "wrappers/mock_session.h"
+
+/*
+ * Check a basic configuration that sets and clears a flag.
+ */
+#define FLAG_TEST(config_param, flag)                                                    \
```

Comment:
```suggestion
#define BOOLEAN_FLAG_TEST(config_param, flag)                                                    \
```

Reply:
Jeremy found a better solution, going with that.